README.md

```diff
@@ -1,112 +1,44 @@
-# Device Metrics Exporter
+# AMD GPU Metrics Exporter
```

Comment:
I belive we want to keep the name Device Metrics Exporter instead of GPU Metrics Exporter because it will eventually expand to include metrics from AI NIC as well, not just GPUs

Reply:
done!